assets/dev/js/editor/elements/views/base.js

```diff
@@ -621,11 +619,7 @@ BaseElementView = BaseContainer.extend( {
 		}
 	},
 
-	renderOnChange( settings ) {
-		if ( ! this.allowRender ) {
-			return;
-		}
-
+	renderChanges( settings, shouldRenderUI = true ) {
```

Comment:
Rename the method, so it will explain what is the returned value (if it returns `true` - what is the meaning of it?)

Reply:
There is no return value, the method just render the changes.

Reply:
https://github.com/elementor/elementor/pull/16252/files#diff-2e887f24662881a1e7a41a616f029a7fac6ca7163b9ae22e5d5e654484270643R657-R663